extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -104,12 +104,17 @@ public function votersModal() {
 		 */
 		$mw =  WallMessage::newFromId( $this->request->getVal( 'id' ) );
 
-		$this->response->setVal('list',
-			$this->app->renderView( 'WallExternalController', 'votersListItems',
-				array( 'from' => 0, 'mw' => $mw, 'id' => $this->request->getVal('id') )
-		));
+		if ( !empty( $mw ) ) {
+			$this->response->setVal('list',
```

Comment:
Since we're touching this code anyways, it'd be nice to clean up this section to follow our whitespace guidelines. We should add additional spacing here, as well as line 110 and line 113.